src/validate.rs

```diff
@@ -0,0 +1,397 @@
+use crate::{
+    config_paths, event,
+    runtime::Runtime,
+    topology::{self, builder::Pieces, Config, ConfigDiff},
+};
+use colored::*;
+use exitcode::ExitCode;
+use futures::compat::Future01CompatExt;
+use std::{fmt, fs::File, path::PathBuf};
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(rename_all = "kebab-case")]
+pub struct Opts {
+    /// Disables topology check
+    #[structopt(long)]
+    no_topology: bool,
+
+    /// Disables environment checks. That includes component checks and health checks.
+    #[structopt(long)]
+    no_environment: bool,
+
+    /// Shorthand for `--no_topology` and `--no_environment` flags. Just `-n` won't disable anything,
+    /// it needs to be used with `t` for `--no_topology`, and or `e` for `--no_environment` in any order.
+    /// Example:
+    /// `-nte` and `-net` both mean `--no_topology` and `--no_environment`
```

Comment:
Nit: the actual flags are `--no-topology` and `--no-environment`.

Reply:
Good catch.